gpuctypes/hip.py

```diff
@@ -5,7 +5,34 @@
 # POINTER_SIZE is: 8
 # LONGDOUBLE_SIZE is: 16
 #
-import ctypes
+import ctypes, sys, os
+
+
+def get_hip(): 
+    try:
+        if 'linux' in sys.platform:
+            return ctypes.CDLL('/opt/rocm/lib/libamdhip64.so')
+        elif 'win' in sys.platform:
+            return ctypes.cdll.LoadLibrary('amdhip64')
+        else:
+            raise RuntimeError('Only windows and linux are supported')
+    except Exception as err:
+        raise Exception('Error: {0}'.format(err))
+
+
+def get_hiprtc():
+    try:
+        if 'linux' in sys.platform:
+            return ctypes.CDLL(os.path.join('/opt/rocm/lib/libhiprtc.so'))
+        elif 'win' in sys.platform:
+            hip_path = os.getenv('HIP_PATH', None)
```

Comment:
Is there a why to not need this? How come it can find amdhip64?

Reply:
`amdhip64` gets installed in System32 where cdll searches for dlls, the rtc does not so have to use `HIP_PATH` to find the hip installation. `HIP_PATH` is added to the env on install of the hip sdk